doc/kconfig/conf.py

```diff
@@ -0,0 +1,62 @@
+# Kconfig documentation build configuration file
+
+import os
+
+NRF_BASE = os.path.abspath(os.path.join(os.path.dirname(__file__), "..", ".."))
+
+# -- General configuration ------------------------------------------------
+
+# Sphinx 2.0 changes the default from 'index' to 'contents'
+master_doc = 'index'
+
+# General information about the project.
+project = 'Kconfig reference'
+copyright = '2019, Nordic Semiconductor'
+author = 'Nordic Semiconductor'
+
+version = '1.0.0'  # Dummy version
```

Comment:
```suggestion
version = '&nbsp;'  # Dummy version
```

Replace with non-breakable space to get rid of the version number, but keep the spacing the same as for the other doc sets.

Reply:
Fixed